packages/console/src/onboarding/pages/SignInExperience/sie-config-templates.ts

```diff
@@ -20,63 +20,39 @@ const configTemplate1: OnboardingSieConfig = {
 };
 
 const configTemplate2: OnboardingSieConfig = {
-  logo: `${assetsUrl}/kvCN1Z2a/logo2.png`,
-  color: '#F47346',
-  identifier: SignInIdentifier.Phone,
-  authentications: [Authentication.Password, Authentication.VerificationCode],
-};
-
-const configTemplate3: OnboardingSieConfig = {
   logo: `${assetsUrl}/IcI0snBP/logo3.png`,
   color: '#FF5449',
   identifier: SignInIdentifier.Username,
   authentications: [Authentication.Password],
 };
 
-const configTemplate4: OnboardingSieConfig = {
+const configTemplate3: OnboardingSieConfig = {
   logo: `${assetsUrl}/7UQyvuFc/logo4.png`,
   color: '#CA4E96',
   identifier: SignInIdentifier.Email,
   authentications: [Authentication.Password],
 };
 
-const configTemplate5: OnboardingSieConfig = {
+const configTemplate4: OnboardingSieConfig = {
   logo: `${assetsUrl}/zB2merH1/logo5.png`,
   color: '#F07EFF',
   identifier: SignInIdentifier.Email,
   authentications: [Authentication.VerificationCode],
 };
 
-const configTemplate6: OnboardingSieConfig = {
-  logo: `${assetsUrl}/CX51jxXS/logo6.png`,
```

Comment:
I think we can keep these templates, since their logo and color branding are still going to diversify the inspiration result. We can only replace phone identifier with other identifiers.

Reply:
Either is fine for me, can change Phone identifier to username identifier.